pom.xml

```diff
@@ -21,6 +21,21 @@
 		<url>https://github.com/camel-tooling/camel-debug-adapter</url>
 	</scm>
 
+	<developers>
+		<developer>
+			<name>Aurélien Pupier</name>
+			<email>apupier@redhat.com</email>
+			<organization>Red Hat</organization>
+			<organizationUrl>https://www.redhat.com</organizationUrl>
+		</developer>
+		<developer>
```

Comment:
I did not do anything in here.

Reply:
you are doing the Pull Request reviews. This is an important thing.